webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -17,12 +24,21 @@ export function mount(component, selector, data, flattenData = false) {
   }
 }
 
-export function mountNode(component, reactNode, data, flattenData = false) {
+export function mountNode(
```

Comment:
This method signature also changes

Reply:
As I mentioned above `mountNode` is for internal use only, and it isn't used elsewhere, when a consumer would like to mount a component he should use only `mount` function, as described in the documentation.